tests/jenkins/TestPublishMinSnapshots.groovy

```diff
@@ -25,7 +25,7 @@ class TestPublishMinSnapshots extends BuildPipelineTest {
 
         helper.registerSharedLibrary(
             library().name('jenkins')
-                .defaultVersion('6.6.1')
+                .defaultVersion('7.3.2')
```

Comment:
Can we add tests to asserting that core plugins are being uploaded?

Reply:
Added now.

Thanks.